dev-tools/reconfigurator-sp-updater/src/main.rs

```diff
@@ -374,8 +377,40 @@ struct SetArgs {
 #[derive(Clone, Debug, Subcommand)]
 enum Component {
     Sp {
+        /// expected version of the active slot
+        #[arg(long, short = 'a')]
+        expected_active_version: ArtifactVersion,
+        /// expected version of the inactive slot
+        #[arg(long, short = 'i')]
+        expected_inactive_version: ExpectedVersion,
+    },
+    Rot {
+        /// whether we expect the "A" or "B" slot to be active
+        #[arg(long, short = 's')]
+        expected_active_slot: RotSlot,
+        /// expected version of the "A" or "B" slot (as per the
+        /// active slot specified in expected_active_slot)
+        #[arg(long, short = 'a')]
         expected_active_version: ArtifactVersion,
+        /// expected version of the "A" or "B" slot (opposite to
+        /// the active slot as specified in expected_active_slot)
+        #[arg(long, short = 'i')]
         expected_inactive_version: ExpectedVersion,
+        /// the expected persistent boot preference written into the current
+        /// authoritative CFPA page (ping or pong)
+        #[arg(long, short = 'b')]
+        expected_persistent_boot_preference: RotSlot,
```

Comment:
Would it make sense to have these default, based on whatever `expected_active_slot` is (presumably the user almost always wants to toggle it)?